components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -1605,4 +1608,93 @@ export class GitpodServerImpl<Client extends GitpodClient, Server extends Gitpod
         return this.termsProvider.getCurrent();
     }
 
+
+    //#region gitpod.io concerns
+    //
+    async adminGetAccountStatement(userId: string): Promise<AccountStatement> {
+        throw new ResponseError(ErrorCodes.EE_FEATURE, `Not implemented in OSS version`);
```

Comment:
We could introduce a separate error code for `SAAS` funtionallity.

```suggestion
        throw new ResponseError(ErrorCodes.SAAS_FEATURE, `Not implemented in this version`);
```

Reply:
Let's go with SaaS

Reply:
done